php/resources/views/subscription.blade.php

```diff
@@ -0,0 +1,151 @@
+<!doctype html>
+<html lang="en">
```

Comment:
This there no base template to eliminate all this repetition?

Reply:
There is, but templating in lumen is kinda ugly:
- https://stackoverflow.com/questions/35798721/how-to-extend-multiple-templates-in-blade-laravel-5


```
// layout.blade.php
<html>
    <head>
        @yield('head')
    </head>
    <body>
        @yield('body')
    </body>
</html>
```
```
// home.blade.php
@extends('layout')

@section('head')
     <title>Hello World</title>
@endsection

@section('head')
     <p>Hello World</p>
@endsection
```

If you want I can refactor to use this?

Reply:
I refactored to templating to make WEB_URL env var easier 👍 😇